syntax = "proto3";
import "payload.proto";

service ContentService {
    rpc setContent (stream Request) returns (Response);
    rpc getContent (Request) returns (Response);
}